### DISCUSSION TITLE
Problem with rosdep and ecl_treads

### DISCUSSION THREAD

**Discussion by pmdSchmidt** [mangdangroboticsclub/minipupper_ros#77] [Q&A] [answered]
Hi,
when executing the command `rosdep install --from-paths src --ignore-src -r -y` i get the error that the rosdep definition for [ecl_threads] is unknown. 
I searched on for an alternative to install ecl_threads, but it seems that there is no version for humble available?!
Do you have an idea how i can still build the package?

Thank you!

**Comment by sskorol** [comment, marked as answer]
As far as I remember, it comes from Champ. I did some refactoring before to adopt Velocity Smoother for Humble. I removed this [dependency](https://github.com/chvmp/champ/pull/115/files#diff-333c005207e08d99248409c55551a7908634955fef288b163f9df617ce21787a) at all.

**Reply by pmdSchmidt** [reply]
Thanks! Removing the dependency solved the problem.
